app/services/roles_service.rb

```diff
@@ -265,7 +275,7 @@ def seed_qa_users!
             end
           end
 
-          puts "Email: #{user.email}\nRoles: #{user.roles.map(&:name)}\n\n"; nil
+          Rails.logger "Email: #{user.email}\nRoles: #{user.roles.map(&:name)}\n\n"
```

Comment:
I'm not positive this will do the same thing it was before, so we should look at it

Reply:
This is one that JP and I googled together HA yes, let's look at it together!